pipelines/gitops-pull-request-rhtap/gitops-pull-request.yaml

```diff
@@ -38,6 +42,10 @@ spec:
           value: $(params.git-url)
         - name: revision
           value: $(params.revision)
+        # We're more interested in remote branches than tags, but this fetches both.
+        # Branch refs are needed for the TARGET_BRANCH feature of gather-deploy-images.
+        - name: fetchTags
```

Comment:
Is it used anywhere?

Reply:
If it fails without fetching them, we do we need a parameter?

Reply:
In this section we're not adding a parameter, we're just passing fetchTags: true to the clone task (the default is false)